### ISSUE TITLE
Problem when using package formidable / node-formidable or anything that includes it, webpack doesn't compile the package correctly

### ISSUE THREAD

**Issue by tcastelli** [AnomalyInnovations/serverless-bundle#42]
As you can see on this thread
https://github.com/node-formidable/node-formidable/issues/337

node-formidable or other packages using it fail to compile correctly this line 
````
if (global.GENTLY) require = GENTLY.hijack(require);
````
creating 

````
var require;if (global.GENTLY) require = GENTLY.hijack(require);
````
this makes that later requires included are not properly transpiled and fail after deploy 
in the line var crypto = require("crypto")

the easiest workaround found in the thread is to add 
```
plugins: [
     new webpack.DefinePlugin({ "global.GENTLY": false })
], 
```

but that doesn't seem to be possible with this plugin. Any chance to allow custom webpack plugins or fix this problem?

**Comment by jayair** [comment]
@tcastelli Hmm that's interesting. We don't have plans to add that option just yet. But from looking at that thread. Do you think they are going to address that issue soon?

**Comment by tcastelli** [comment]
No idea tbh, I just forked this project and added the missing plugin and all is working for now.  It's a shame that there's no way to customize this plugin a little more. I guess this plugin could look for a json file path and add those external options to webpack easily.

**Comment by jayair** [comment]
Yeah I'll keep this open. If more people run into this or something similar we'll fix it.

**Comment by rodrigogs** [comment]
I have the same problem and I don't want to create a lambda layer just to have a single dependency working :/

**Comment by jayair** [comment]
We have a specific fix for Formidable — https://github.com/AnomalyInnovations/serverless-bundle#package-specific-config
